NOTES.md

```diff
@@ -456,4 +412,4 @@ Edge:
 console.time({ toString() { throw new Error("Farolino") } }) // toString called, no error thrown, no timer started
 ```
 
-Bug: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11201116/
\ No newline at end of file
+Bug: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11201116/
```

Comment:
I honestly don't know what to make of this line diff. Maybe a platform-specific EOF or line break change?

Reply:
Got it. Anyways, I'd like for you to take a look at this and let me know what you think